src/main/java/games/strategy/triplea/delegate/DiceRoll.java

```diff
@@ -55,8 +56,7 @@
   // this does not need to match the Die with isHit true
   // since for low luck we get many hits with few dice
   private int m_hits;
-
-
+  private double m_expectedHits;
```

Comment:
I think this line is triggering a Checkstyle violation.

I know we've tried to be consistent with naming in `Serializable` classes in the past, even if it introduces new Checkstyle violations.  However, note that this class is `Externalizable`, and the field names are not serialized.  I've been fixing such naming violations in `Externalizable` classes recently, and just haven't gotten to this one yet.

What I'm saying is that you can probably just name this field `expectedHits`, as all the other fields will have their `m_` prefix removed soon.

Reply:
Good point. Renamed.